Tests/XcbeautifyLibTests/XcbeautifyLibTests.swift

```diff
@@ -320,4 +320,19 @@ final class XcbeautifyLibTests: XCTestCase {
 
     func testWriteFile() {
     }
+
+    func testPackageGraphResolved() {
+
+        // Start
+        let start = noColoredFormatted("Resolve Package Graph")
+        XCTAssertEqual(start, "Resolve Package Graph")
+
+        // Ended
+        let ended = noColoredFormatted("Resolved source packages:")
+        XCTAssertEqual(ended, "Resolved source packages")
```

Comment:
Should we include the trailing colon too?

Reply:
`xcodebuild` produces a line with a colon at the end. Do you mean adding it to the formatted output?

Reply:
Yes. There will be contents after this, right?